bin/integration-tests/build.rs

```diff
@@ -13,6 +13,58 @@ use std::{env, fs};
 
 const TEST_PREFIX: &str = "test_";
 
+// Template constants - embedded directly in build script for portability
```

Comment:
nit: something like this maybe?
```suggestion
// GENERATED FILE TEMPLATES
// ================================================================================================
```